DependencyInjection/DoctrinePHPCRExtension.php

```diff
@@ -71,6 +71,7 @@ public function load(array $configs, ContainerBuilder $container)
 
         if (!empty($config['manager_registry_service_id'])) {
             $container->setAlias('doctrine_phpcr', new Alias($config['manager_registry_service_id']));
+            $container->getDefinition($config['manager_registry_service_id'])->setPublic(true);
```

Comment:
why do we dynamically set this service to public? i think the manager registry service should be always public

Reply:
from Symfony 3.3 on all services are private by default, that is the major reason and work for the mirgration

Reply:
and i miss some more, all that i do public in: https://github.com/symfony-cmf/testing/pull/145 really hard to find :-)

Reply:
I understand that, but setting it hard to public feels like having the behavior the user used to have before Symfony 3.3, or?

Reply:
A now i see that i changed the wrong line ... thanks will change it.

Reply:
yep, the alias should be public, but not the other service. i tried to find what this even is for, and found no single bit of documentation neither in the doctrine phpcr bundle config reference, nor in the orm bundle doc...

i think what you need to make public however is `<service id="doctrine_phpcr" class="%doctrine_phpcr.class%">` in phpcr.xml. (i will do a PR to eliminate the class parameters for master (upcoming version 2) like we do everywhere else)

Reply:
> I understand that, but setting it hard to public feels like having the behavior the user used to have before Symfony 3.3, or?

Private services have existed ever since. You don't know whether a service is private due to the changed defaults in Symfony 4 or because it was set to private explicitly. It's not your responsibility to create BC for other people's services.